### PR TITLE
5049 fix sse npe

### PR DESCRIPTION
**Pull request by WhiteCat22**
https://github.com/OpenLiberty/open-liberty/issues/5049

This PR reverts #2084 and performs a null check in MessageContextImpl.java instead.

### CODE REVIEW

**Comment by WhiteCat22** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/ext/MessageContextImpl.java [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+                m.getExchange().put("cxf.io.cacheinput", Boolean.FALSE);
+                throw new WebApplicationException(e, 413);
+            } catch (HeaderSizeExceededException e) {
+                throw new WebApplicationException(e, 413);
+            }
+        }
+        if (keyValue.equals("WRITE-" + Message.ATTACHMENTS)) {
+            // Liberty Change Start - #5049
+            if (m.getExchange().getOutMessage() == null) {
+                return null;
```
Comment (andymc12) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/ext/MessageContextImpl.java]: Hmm... if we return null here, it seems like we might be preventing this method from returning something that might be on the "in" message from the code below - what if we did something like:
```
if (keyValue.equals("WRITE-" + Message.ATTACHMENTS) && m.getExchange().getOutMessage() != null) {
    return m.getExchange().getOutMessage().get(Message.ATTACHMENTS);
```

**Comment by WhiteCat22** [comment]
#build try not returning null

**Comment by WhiteCat22** [comment]
build successful in both cases. We are going to err on the side of caution and not return null when there is no outMessage, but continue through MessageContextImpl.get().

**Pull request merged**